Cargo.toml

```
@@ -18,7 +18,12 @@ exclude = ["/.*"]
 bench = false
 
 [dependencies]
+cfg-if = "1"
 crossbeam-utils = { version = "0.8.11", default-features = false }
+portable-atomic = { version = "0.3", default-features = false, optional = true }
+
+[target.'cfg(loom)'.dependencies]
+loom = "0.5"
```

Some people complain when loom appears in Cargo.lock (even if it never actually builds).
(https://github.com/crossbeam-rs/crossbeam/pull/487#issuecomment-783365041, https://github.com/crossbeam-rs/crossbeam/issues/665, https://github.com/tokio-rs/bytes/issues/411, https://github.com/tokio-rs/bytes/issues/400, etc.)
https://github.com/crossbeam-rs/crossbeam/pull/666

To avoid them, I tend to prefer to use cfg-ed optional deps like https://github.com/crossbeam-rs/crossbeam/pull/666.

```suggestion
loom = { version = "0.5", optional = true }
```